src/server/database/documents/Life.ts

```diff
@@ -0,0 +1,11 @@
+import { ObjectId } from 'mongodb';
+
+export type Life = {
+    _id: ObjectId;
+    firstName: string;
+    lastName: string;
+    birthday: Date;
+    title: string;
+    description: string;
+    hobbies: string[];
+};
```

Comment:
your EOF is not correct, there should be only one empty last line

Reply:
Fixed. Will include in next commit (part 4)